linode/helper/unit/framework_util.go

```diff
@@ -0,0 +1,41 @@
+package unit
```

Comment:
Do we already have a package for unit test utils, and if not is there a better place for this package to live?